e2e/cypress/integration/specs/organizationmanagement/user-management-browsing.b2b.e2e-spec.ts

```diff
@@ -29,13 +31,22 @@ describe('User Management', () => {
     });
   });
 
+  it('should show budget popover on progress bar mouseover', () => {
```

Comment:
was unable to run the tests, this might not work :/

Reply:
this part seems to be working just fine 😃

Reply:
![image](https://user-images.githubusercontent.com/23452927/86437497-f2d84500-bd04-11ea-8a66-3fef25bfc82a.png)

further down below, you are using `have.text` (exact match), but there are spaces around the price. you can use `contain` instead. for the $10000 budget, you'll have to do some number formatting before, or just compare with a pre-formatted string.